lib/cbus_elixir/app/meeting.ex

```diff
@@ -2,11 +2,12 @@ defmodule CbusElixir.App.Meeting do
   use Ecto.Schema
   import Ecto.Changeset
   import Ecto.Query
-  alias CbusElixir.App.{Meeting, Speaker}
+  alias CbusElixir.App.{ Attendee, Meeting, Speaker}
```

Comment:
This is a preference, but I prefer to have alias' split out per line. So:
```elixir
alias CbusElixir.App.Attendee
alias CbusElixir.App.Meeting
alias CbusElixir.App.Speaker
```

This way you can do an easy text search to find module usage

Reply:
I can adjust them, makes no difference to me.